src/ResponseValidator.ts

```diff
@@ -87,6 +87,10 @@ export class ResponseValidator {
             this._validateIdTokenAttributes(response, state.id_token);
             logger.debug("ID Token validated");
         }
+        else if (state.id_token) {
```

Comment:
Maybe ts better to pass user.profile into state and reuse that, as it might contains more info than id_token. Profile is a combination of the `id_token` and the user info endpoint...